### ISSUE TITLE
Widgets does not show up in Widget Inspector

### ISSUE THREAD

**Issue by EmreSURK** [iampawan/VelocityX#96]
When building a UI using VelocityX, Widget Inspector doe not show any widgets.  

A piece of code to test: 

```
  import 'package:flutter/material.dart';
  import 'package:velocity_x/velocity_x.dart';
  
  class StandartScreen extends StatelessWidget {
    @override
    Widget build(BuildContext context) {
      return Scaffold(
        body: "text".text.make().box.make().centered(),
      );
    }
  }
```

There two screenshots to explain:
Widget Inspector if we use Velocity: 
<img width="1769" alt="current" src="https://user-images.githubusercontent.com/9450430/118534098-86fe9f80-b751-11eb-9f47-c0f359682a90.png">

Widget Inspector without Velocity: 
<img width="1667" alt="expected" src="https://user-images.githubusercontent.com/9450430/118534158-97af1580-b751-11eb-97cc-d682577b7fc7.png">

**Expected behavior**
Widget Inspector should show all Widgets. 

Thanks.

**Comment by iampawan** [comment]
It's a known issue. Not sure what's the exact reason for this though.

**Comment by ghost** [comment]
Maybe try recreating the bug in a throwaway project by copying some subset of the project to it and then removing pieces one by one until what causes bug can be found.

**Comment by iampawan** [comment]
Fixed with Flutter 3.0
